sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/samconfig.toml

```diff
@@ -0,0 +1,12 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "sdlf-dev-unzip-fargate"
+region = "us-east-1"
```

Comment:
Same comment as above in case the region changes

Reply:
This file is created when deploying the SAM template using the parameters that you configure when calling the "deploy.sh" script (lines 127 - 137 of deploy.sh) so this file is just garbage from older deployments